### ISSUE TITLE
Intermittent AuthAPiError for Okta org openid-configuration Loading

### ISSUE THREAD

**Issue by MultiPlanJSEval** [okta/okta-auth-js#609]
Intermittently getting below shown error in Chrome, FireFox, Microsoft Edge browsers. We use     "@okta/okta-auth-js": "^4.6.1",
    "@okta/okta-react": "^4.1.0", components,

![image](https://user-images.githubusercontent.com/39303958/106194297-3cdab980-6174-11eb-9252-51dd05ea1274.png)

**Comment by shuowu** [comment]
It looks like a CORS issue, have you added your localhost (http://localhost:5280) to the trusted origins?

You can add it by following API -> Trusted Origins -> Add origin in your OKTA dashboard.

**Comment by MultiPlanJSEval** [comment]
Yes we did configured CORS in okta org otherwise it will not work most of the times. As I said, this error is intermittent.

**Comment by shuowu** [comment]
Correct. From the error log, I cannot tell the exact cause of the issue. 

Meanwhile, I suggest you can reach out to our [support team](https://support.okta.com/help/s/?language=en_US), they can help dive into details about your particular case.

**Comment by luckyluckytina** [comment]
@MultiPlanJSEval  Can you please check the below things? 1) Does the chrome incognito window work? 2) In the normal chrome window, check if you have Okta Browser plugin installed. If that case, try to test with plugin disabled and see if it will work.
